sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2516,4 +2516,13 @@ class HiveDDLSuite
       }
     }
   }
+
+  test("SPARK-28443: fail the DDL command if it creates a table with null-type columns") {
```

Comment:
BTW do we need this test? I think https://github.com/apache/spark/pull/25198/files#diff-85f240f8452fa64de6671e41801fe68fR550 is good enough.

Reply:
I forget that spark not support `create table t as select null as c` now. Remove this.